pkg/controller/clustersync/clustersync_controller.go

```diff
@@ -928,8 +928,9 @@ func (r *ReconcileClusterSync) setFirstSyncSetsSuccessTime(syncStatuses []hivein
 			lastSuccessTime = status.FirstSuccessTime
 		}
 	}
-	clusterSync.Status.FirstSyncSetsSuccessTime = lastSuccessTime
+	clusterSync.Status.FirstSuccessTime = lastSuccessTime
 	allSyncSetsAppliedDuration := lastSuccessTime.Time.Sub(cd.Status.InstalledTimestamp.Time)
+	logger.Debugf("observed syncsets applied duration: %v seconds", allSyncSetsAppliedDuration.Seconds())
```

Comment:
This should be Infof, it's a one time statement and could be important into to be able to see in the logs. Most Hive shards run at Info.